packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -1,12 +1,22 @@
 import { ContextualMenuItemType, IContextualMenuItem } from "@fluentui/react";
 import * as React from "react";
 import { useDispatch, useSelector } from "react-redux";
+import { shell } from "electron";
 
 import useOpenWithMenuItems from "./useOpenWithMenuItems";
 import FileDetail from "../entity/FileDetail";
 import FileFilter from "../entity/FileFilter";
 import { interaction, selection } from "../state";
 
+/**
+ * Reveals a selected file in the user's native file browser
+ */
+const openNativeFileBrowser = (fileDetails: FileDetail) => {
+    if (fileDetails.localPath) {
+        shell.showItemInFolder(fileDetails.localPath);
```

Comment:
This will likely need to be dispatched as an action to the redux state where a side-effect would occur that would call the `FileViewerService` which would do this functionality in a platform specific way. For web it would just throw a not implemented error.

Reply:
Good point! Addressed in 0b8c222201b63d8dd6c926f1a52f6e51f39cd10d and 0b8c222201b63d8dd6c926f1a52f6e51f39cd10d